.github/workflows/test.main.kts

```diff
@@ -3,12 +3,14 @@
 @file:DependsOn("io.github.typesafegithub:github-workflows-kt:3.3.0")
 @file:DependsOn("it.krzeminski:snakeyaml-engine-kmp:3.1.1")
 @file:DependsOn("io.github.optimumcode:json-schema-validator-jvm:0.5.1")
+@file:DependsOn("com.github.sya-ri:kgit:1.1.0")
```

Comment:
Thanks for using the type-safe approach, also: I learned there's https://github.com/sya-ri/KGit 😄 I've always used jgit (like [here](https://github.com/typesafegithub/github-actions-typing-catalog/blob/db76dd0f1149901e1cdf60ec98d568b32fa7eb71/.github/workflows/update-metadata.main.kts#L5)) or grgit.